third_party/terraform/tests/resource_pubsub_topic_test.go

```diff
@@ -2,21 +2,26 @@ package google
 
 import (
 	"fmt"
+	"strings"
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
 )
 
 func TestAccPubsubTopic_update(t *testing.T) {
 	t.Parallel()
 
 	topic := fmt.Sprintf("tf-test-topic-%s", acctest.RandString(10))
+	providers := getTestAccProviders(t.Name())
```

Comment:
Does it make sense to add a wrapper around resource.Test that would handle some of these things that would need to be added to all tests?

Reply:
That's a good idea. I'll have to modify the calls to `acctest.RandString()` to allow for seeding based on a value stored alongside the cassette as well, so that could go in there too.